api/src/job.js

```diff
@@ -239,15 +271,18 @@ class Job {
                 }
             });
 
+            // when the process writes to stdout, send the data to the event_bus (over websocket later)
             proc.stdout.on('data', async data => {
                 if (event_bus !== null) {
                     event_bus.emit('stdout', data);
-                } else if ((stdout.length + data.length) > this.runtime.output_max_size) {
+                } else if (
+                    stdout.length + data.length >
```

Comment:
Same this as before with the brackets.

Reply:
Thank for the review @HexF 
The added curly braces must are unintended formatting mistake, I'll fix that